crates/ty_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -13,6 +13,12 @@
 //!   declarations; annotated function parameters are also declarations. We consider `def` and
 //!   `class` statements to also be declarations, so as to prohibit accidentally shadowing them.
 //!
+//! * A "place" is semantically a location where a value can be read or written,
+//!   and syntactically an expression that can be placed on the left-hand side of an assignment,
```

Comment:
Similar here, maybe:
```suggestion
//!   and syntactically, an expression that can be the target of an assignment,
```